source/_docs/energy/electricity-grid.markdown

```diff
@@ -31,11 +31,13 @@ We have worked with creator [Marcel Zuidwijk](https://www.zuidwijk.com) to devel
 
 ![Photo of SlimmeLezer attached to a smart electricity meter](/images/docs/energy/slimmelezer.jpg)
 
-#### Connect via Zigbee Energy Profile
+#### Connect to devices that uses Zigbee Smart Energy Standard
 
-The Zigbee Energy Profile is a wireless energy standard to provide real-time information about electricity usage. This standard is available in some meters in the US, UK and Australia. This is not "normal" Zigbee as implemented by Home Assistant but requires special certified hardware.
+[Zigbee Smart Energy Standard](https://csa-iot.org/all-solutions/smart-energy/) is a wireless protocol designed by the CSA (Connectivity Standards Alliance, formerly the Zigbee Alliance) to specifically target industrial and commercial utilities to enable real-time monitoring of energy consumption and actively managing energy consumption at the end-user level. This wireless protocol is supported by some utility meters for monitoring electricity, gas, and water consumption in the US, EU, UK, and Australia.
```

Comment:
This is way too wordy for this page. Nobody cases about the CSA or how it was formally called. Let's remove it and be a little bit more condensed about the message to send.